src/pages/Account/Tabs/ModulesTab.tsx

```diff
@@ -55,19 +80,78 @@ function ModulesContent({
   );
 }
 
-type ModulesTabProps = {
+function ModulesContent2({address}: {address: string}): JSX.Element {
```

Comment:
lol lets think of a better name for this function?

Reply:
let me change it to ModulesContentReworked s.t. its purpose is clear